auth/src/main/java/com/firebase/ui/auth/util/CredentialsApiHelper.java

```diff
@@ -34,80 +27,28 @@
 import com.google.android.gms.tasks.Task;
 import com.google.android.gms.tasks.TaskCompletionSource;
 
-import java.util.ArrayList;
-import java.util.List;
-
 /**
  * A {@link com.google.android.gms.tasks.Task Task} based wrapper for the Smart Lock for Passwords
  * API.
  */
 public class CredentialsApiHelper {
-
     @NonNull
     private final GoogleApiClientTaskHelper mClientHelper;
 
     private CredentialsApiHelper(GoogleApiClientTaskHelper gacHelper) {
         mClientHelper = gacHelper;
     }
 
-    public CredentialRequest createCredentialRequest(List<IdpConfig> providers) {
```

Comment:
This is the only non move stuff around change: a few of these methods weren't being used so I removed them.